third_party/yarr/YarrInterpreter.cpp

```diff
@@ -63,16 +137,16 @@ class Interpreter {
 
     struct DisjunctionContext {
         DisjunctionContext()
-            : term(0)
         {
+            // member variables are inited later
```

Comment:
How about initializing `term` member with nullptr?
Uninitialized member would incur warnings by code analyzer tools later.

Reply:
Adding init expression cause performance effect. because it will be inited later with another values.